docs/api/ipc-main.md

```diff
@@ -88,6 +88,54 @@ Removes the specified `listener` from the listener array for the specified
 
 Removes listeners of the specified `channel`.
 
+### `ipcMain.handle(channel, listener)`
+
+* `channel` String
+* `listener` Function
+  * `event` IpcMainEvent
```

Comment:
```suggestion
  * `event` IpcMainHandleableEvent
```

We should keep `reply` on the `IpcMainEvent` interface, there are use cases for that method (multiple responses, responding with a different channel, etc.)